components/soc/esp32/include/soc/emac_ext_struct.h

```diff
@@ -11,7 +11,7 @@ extern "C" {
 
 #include <stdint.h>
 
-typedef volatile struct emac_ext_dev_t {
+typedef volatile struct {
```

Comment:
Alternatively we can only add the `volatile` to each member

```suggestion
typedef struct emac_ext_dev_t {
    volatile union {
        struct {
            uint32_t div_num : 4;
            uint32_t h_div_num : 4;
            uint32_t dly_num : 2;
            uint32_t reserved10 : 22;
        };
        uint32_t val;
    } ex_clkout_conf;
    ....
```

Reply:
cc @kostaond ^^